.github/workflows/release-artifacts.yml

```diff
@@ -89,17 +89,62 @@ jobs:
           name: debs
           path: debs/*
 
-  build-sdist:
-    name: "Build pypi distribution files"
-    uses: "matrix-org/backend-meta/.github/workflows/packaging.yml@v1"
+  build-wheels:
+    name: Build wheels on ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-20.04, macos-10.15]
+        is_pr: [${{ startsWith(github.ref, 'refs/pull/') }}]
+
+        exclude:
+          # Don't build macos wheels on PR CI.
+          - is_pr: true
+            os: "macos-10.15"
```

Comment:
Do we have anything to alert us if the macos wheels fail to build? I feel like simply having the green tick turn into a red cross on https://github.com/matrix-org/synapse/tree/develop is easy to miss, so I would say ideally we would need something like we have for the twisted trunk job.

Reply:
Mmm, maybe we should just have a generic issue if the develop / release tests fail? Rather than special casing each one.

Note we already only tests small subsets of debian version when building debs on PRs, so this is in line with that.